examples/platform/silabs/display/lcd.cpp

```diff
@@ -149,11 +147,102 @@ void SilabsLCD::WriteDemoUI()
     }
 }
 
+void SilabsLCD::WriteStatus()
+{
+    uint8_t lineNb = 0;
+    char str[20];
+
+    GLIB_clear(&glibContext);
+    sprintf(str, "# Fabrics : %d", mStatus.nbFabric);
+    GLIB_drawStringOnLine(&glibContext, str, lineNb++, GLIB_ALIGN_LEFT, 0, 0, true);
+
+    if (strlen(mStatus.networkName) >= sizeof(str))
```

Comment:
You could use chip::Platform::CopyString here and not have extra logic for sizeof.

Reply:
Will investigate in follow up